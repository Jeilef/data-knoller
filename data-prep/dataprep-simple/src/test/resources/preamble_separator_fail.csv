1,2,3,4
4,3,2,2
13,52,3
#this,is,a,preamble
#preamble
#preamble
34,2,1,5
,4,5,2,5,2,5,7
#this,is,
#a,preamble
#postamble