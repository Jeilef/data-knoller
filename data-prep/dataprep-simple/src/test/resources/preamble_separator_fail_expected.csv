1,2,3,4
4,3,2,2
13,52,3
34,2,1,5
,4,5,2,5,2,5,7